### DISCUSSION TITLE
How to constrain corepack version

### DISCUSSION THREAD

**Discussion by simPod** [renovatebot/renovate#27452] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.202

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

Corepack is currently broken https://github.com/nodejs/corepack/issues/379 and I'm getting 0.25.1 version installed. I'd like to constrain it into 0.24.1. 
I've found there should be a way but I don't know where to set it https://github.com/renovatebot/renovate/commit/d9a24c5d960a9052ac4b79f08123d13de259b67c

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Set constraints.corepack in Renovate config
